Gems/AudioSystem/Code/Source/AudioSystemGemSystemComponent.cpp

```diff
@@ -98,11 +102,22 @@ namespace AudioSystemGem
 
     void AudioSystemGemSystemComponent::Init()
     {
-        m_loseFocusRequest.nFlags = Audio::eARF_PRIORITY_HIGH;
-        m_loseFocusRequest.pData = &m_loseFocusData;
+    #if defined(AUDIO_SYSTEM_EDITOR)
+        m_cameraTransformHandler = AZ::RPI::MatrixChangedEvent::Handler(
+            []([[maybe_unused]] const AZ::Matrix4x4& matrix)
+            {
+                auto atomViewportRequests = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
+                auto viewportContext = atomViewportRequests->GetDefaultViewportContext();
 
-        m_getFocusRequest.nFlags = Audio::eARF_PRIORITY_HIGH;
-        m_getFocusRequest.pData = &m_getFocusData;
+                if (auto audioSystem = AZ::Interface<Audio::IAudioSystem>::Get();
+                    audioSystem != nullptr && viewportContext != nullptr)
+                {
+                    Audio::ListenerRequest::SetWorldTransform setWorldTM;
+                    setWorldTM.m_transform = viewportContext->GetCameraTransform();
+                    audioSystem->PushRequest(AZStd::move(setWorldTM));
+                }
+            });
+    #endif // AUDIO_SYSTEM_EDITOR
```

Comment:
Well I see why the dependency on Atom RPI Public exist now.
Is the dependency on AZ::AtomCore needed however?

Reply:
I was running into an issue with only RPI Public as dependency.  I can try removing AtomCore and see what that is and post it here.  Maybe it exposes something that could be decoupled from AtomCore.

Reply:
Here's the error I was getting:
```
3>D:\code\o3de\Gems\Atom\RPI\Code\Include\Atom/RPI.Public/Buffer/BufferSystemInterface.h(15,10): fatal error C1083: Cannot open include file: 'AtomCore/Instance/Instance.h': No such file or directory
3>D:\code\o3de\Gems\Atom\RPI\Code\Include\Atom/RPI.Public/Buffer/BufferSystemInterface.h(15,10): fatal error C1083: #include <AtomCore/Instance/Instance.h>
3>D:\code\o3de\Gems\Atom\RPI\Code\Include\Atom/RPI.Public/Buffer/BufferSystemInterface.h(15,10): fatal error C1083:          ^
3>Done building project "AudioSystem.Editor.Static.vcxproj" -- FAILED.
```

Reply:
The `Atom_RPI.Public` target lists `AZ::AtomCore` as a **PRIVATE** `BUILD_DEPENDENCY`.  Moving it to be a **PUBLIC** `BUILD_DEPENDENCY` does fix it.

Any Atom contributors want to weigh in on this potential change?

Reply:
Seems OK to me,  pinging @VickyAtAZ @dmcdiarmid-ly @antonmic

Reply:
I think it is fine to add a Public BUILD_DEPENDENCY